### PR TITLE
Modify delete command

### PR DESCRIPTION
**Pull request by yuehernkang**

### CODE REVIEW

**Comment by yuehernkang** [comment]
closes #33

**Review by zacchaeuschok — APPROVED** [review]
LGTM

File: src/test/java/seedu/condonery/logic/commands/DeleteCommandTest.java [hunk 1]
```diff
@@ -28,10 +28,10 @@ public class DeleteCommandTest {
 
     @Test
     public void execute_validIndexUnfilteredList_success() {
-        Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
+        Person personToDelete = model.getFilteredPropertyList().get(INDEX_FIRST_PERSON.getZeroBased());
```
Comment (misterpuffin) [inline src/test/java/seedu/condonery/logic/commands/DeleteCommandTest.java:31]: I think shld rename to property!

File: src/test/java/seedu/condonery/logic/commands/DeleteCommandTest.java [hunk 2]
```diff
@@ -41,20 +41,20 @@ public void execute_validIndexUnfilteredList_success() {
...
+        assertCommandFailure(deleteCommand, model, Messages.MESSAGE_INVALID_PROPERTY_DISPLAYED_INDEX);
     }
 
     @Test
     public void execute_validIndexFilteredList_success() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
 
-        Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
+        Person personToDelete = model.getFilteredPropertyList().get(INDEX_FIRST_PERSON.getZeroBased());
         DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST_PERSON);
```
Comment (misterpuffin) [inline src/test/java/seedu/condonery/logic/commands/DeleteCommandTest.java:55]: same here!

File: src/test/java/seedu/condonery/logic/commands/EditCommandTest.java [hunk 3]
```diff
@@ -72,7 +72,7 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
     @Test
     public void execute_noFieldSpecifiedUnfilteredList_success() {
         EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, new EditPersonDescriptor());
-        Person editedPerson = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
+        Person editedPerson = model.getFilteredPropertyList().get(INDEX_FIRST_PERSON.getZeroBased());
```
Comment (misterpuffin) [inline src/test/java/seedu/condonery/logic/commands/EditCommandTest.java:75]: this file also!

**Review by misterpuffin — APPROVED** [review]
will approve first but pls rename before merging! thankss

**Pull request closed without merging**